enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -60,424 +55,481 @@ recommended to review it before proceeding.
+and Christmas due to peak shopping, however, other clusters may have low usage during that time and is an ideal period
+for deployments.
+
+To control these groups, the user will interact with a new Kubernetes resource of `Rollout` as shown below.
+
+```yaml
+apiVersion: v1beta1
+kind: Rollout
+metadata:
+  name: policy-<same as root policy>
+  namespace: <same as root policy>
+spec:
+  approvalsForVersion: "2.0" <-- optional if the user wants to tie the approval to a specific policy "version".
+  decisionGroups:
+    - groupName: dev
+      rolloutApproved: true <-- user sets this to true when ready
+    - groupName: uat:
+      rolloutApproved: false
+    - groupName: prod
+      rolloutApproved: false
+  ungrouped:
+    rolloutApproved: false
+```
 
-  (For technical notes see [Development Phase 1](#development-phase-1))
+This control is separate from the policy's `spec.rolloutStrategy` because the `status` field is required for the
+technical implementation as described in another section. Having it separate also allows control of the rollout in a
+console or CLI even if the policy is managed by GitOps. Additionally, if the policy applies to multiple hubs, the groups
+may be different on those hubs.
+
+The name, with a `policy-` prefix, and the namespace of the `Rollout` needs to be the same as the root policy it refers
+to. This object will be automatically created by the Governance Policy Propagator with only the `status` and
+`metadata.ownerReferences` fields set for storing state.
+
+The `spec.decisionGroups` field is set by the user to indicate which groups are approved. These groups directly match
+those in the [decisionStrategy](https://open-cluster-management.io/concepts/placement/#decision-strategy) field in the
+`Placement` API. If the user wants a subset of the groups to be automatically approved, they would add the groups to the
+policy's `spec.rolloutStrategy.manualPerGroup.mandatoryDecisionGroups` array.
+
+If the user wants the approval to be conditional on a policy version, the user must set `spec.approvalsForVersion` to a
+value that matches the `policy.open-cluster-management.io/version` annotation on the root policy. By default, the
+approvals apply to all policy versions.
+
+If the decision groups do not capture all clusters in the placement predicate, the user can set
+`spec.ungrouped.rolloutApproved` to `true` to approve those clusters.
+
+### Status
+
+#### Replicated Policy Status
+
+A replicated policy is the policy copy in the managed cluster namespace. Each replicated policy will have a rollout
+status defined as follows:
+
+| Status          | Description                                                                                           |
+| --------------- | ----------------------------------------------------------------------------------------------------- |
+| `ToApply`       | The cluster is waiting for the new version of the policy but the old version is still applied.        |
```

Comment:
hmm to me, await , ready  are better` to apply` not clear to me

Reply:
This enum has been out there for a little bit, so I suspect there may be opposition to updating this particular one.